reverse-strings/go.mod

```diff
@@ -0,0 +1,8 @@
+module	github.com/ivansukach/go-internship/reverse-strings
+
+go 1.16
+
+require (
+	github.com/google/go-cmp v0.5.5
+	golang.org/x/exp/errors v0.0.0-20210503015746-b3083d562e1d
```

Comment:
We can get rid of this modules since we don't use them anymore
Make sure go.sum will be empty as well